internal/confmapprovider/discovery/README.md

```diff
@@ -145,6 +164,7 @@ SPLUNK_DISCOVERY_EXTENSIONS_k8s_observer_ENABLED=false
 The priority order for discovery config content from lowest to highest is:
```

Comment:
This doesn't include bundle.d on purpose?

Reply:
Good call, added.